src/engine/enginemaster.h

```diff
@@ -248,6 +248,12 @@ class EngineMaster : public QObject, public AudioSource {
     CSAMPLE m_headphoneMasterGainOld;
     CSAMPLE m_headphoneVolumeOld;
 
+    const QString m_masterGroup;
```

Comment:
sure I can do that -- there's only one EngineMaster so it doesn't matter much though